File: app/src/androidTest/java/com/example/task2/models/benchmarks/FakeCollectionBenchmark.java
package com.example.task2.models.benchmarks;

import static com.example.task2.Constants.RESULT_COLLECTION;
import static com.example.task2.Constants.SLEEP_TIME_FOR_MODELS;

public class FakeCollectionBenchmark extends CollectionBenchmark implements Benchmark {

    @Override
    public long measureTime(CellOperation item, int number) {
        try {
            Thread.sleep(SLEEP_TIME_FOR_MODELS);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        return RESULT_COLLECTION;
    }
}

File: app/src/androidTest/java/com/example/task2/models/benchmarks/FakeCollectionBenchmark.kt
package com.example.task2.models.benchmarks

import com.example.task2.Constants

class FakeCollectionBenchmark : CollectionBenchmark(), Benchmark {

    override fun measureTime(item: CellOperation, number: Int): Long {
        try {
            Thread.sleep(Constants.SLEEP_TIME_FOR_MODELS)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }
        return Constants.RESULT_COLLECTION
    }
}
